src/views/authenticated/List.tsx

```diff
@@ -21,13 +21,23 @@ export function List({ data: unfilteredListItems, listPath }: Props) {
 			.sort(comparePurchaseUrgency);
 	}, [searchTerm, unfilteredListItems]);
 
+	const Header = () => {
+		return (
+			<p>
+				Hello from the <code>/list</code> page!
+			</p>
+		);
+	};
+
+	if (!listPath) {
```

Comment:
I see that the `Header` handles the retruning early if `listPath` is null or falsy so that the components that require `listPath` to be defined don't have to do the checks themselves...

I think I am trying to wrap my head around why do that check here in the `List` & `ManageList` instead of higher in the `App`, would it have over complicated the logic or does it make conceptual sense to put it here for both of them?

Reply:
We probably could have made those routes re-direct if `listPath` was `null`, but that would be changing existing behavior in the app. We've been displaying these routes + header if there was no `listPath`, and I didn't want to take the liberty of changing that because I think that should be the team's decision, and could be thought of as part of the project you had mentioned some interest in; there's a good amount of overlap between "what does this look like when you're not-signed in" and "what does it look like when you first sign in"

Reply:
That makes total sense, I didn't think about how we have had specific functionality and how that could change the app. Thinking about how the overall process/usage flow of the app is a consideration in where the `listPath` can be null and can't be.